Intrepid.podspec

```diff
@@ -25,7 +25,7 @@ Pod::Spec.new do |s|
   s.source        = { :git => "https://github.com/IntrepidPursuits/swift-wisdom.git", :tag => "#{s.version}" }
   s.exclude_files = "tests/**/*"
   s.platform      = :ios
-  s.ios.deployment_target = "8.0"
+  s.ios.deployment_target = "9.0"
```

Comment:
Does this have to change? I don't think we should bump target requirements unless required by the API's we're using. Seems like `compactMap` works on 8.0, still as long as it's compiled in xcode 9.3.

Reply:
The spec bump was more to handle the `NSLayoutAnchors` usage that was introduced through PR #147

Reply:
Ok, understood ⚓️